openfisca_country_template/reforms/modify_social_security_brackets.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# This file defines a reform.
+# A reform is a set of modifications to be applied to a reference tax and benefit system to carry out experiments.
+# See https://doc.openfisca.fr/reforms.html
+
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+
+
+class modify_social_security_brackets(Reform):
```

Comment:
This describes what is done, and that's not usually how reforms are named: they usually are named after the law project, or the impact they want to have. These examples are meaningful only if one can rely to it on business cases.

So, here, what would have happened in the modelled country? Refer to the README and add to the story there: maybe this reform was a law proposal made by some representatives as an alternative to the universal basic income?